File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SchsSCH0201Q04ReserTimeListInfo.java
package nta.med.data.model.ihis.schs;

import java.util.Date;

/**
 * @author Administrator
 *
 */
/**
 * @author Administrator
 *
 */
public class SchsSCH0201Q04ReserTimeListInfo {
	
	private String hangmogCode;
	private String hangmogName;
	private Date reserDate;
	private String reserTime;
	private String suname;
	private String inputId ;
	private String inOutGubun;
	private String inputGwa;
	private String sex ;
	private Integer age ;
	private Date actingDate;
	private String comments;
	private String bunho;
	private String doctor ;
	private String updName ;
	public SchsSCH0201Q04ReserTimeListInfo(String hangmogCode,
			String hangmogName, Date reserDate, String reserTime,
			String suname, String inputId, String inOutGubun, String inputGwa,
			String sex, Integer age, Date actingDate, String comments,
			String bunho, String doctor, String updName) {
		super();
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.reserDate = reserDate;
		this.reserTime = reserTime;
		this.suname = suname;
		this.inputId = inputId;
		this.inOutGubun = inOutGubun;
		this.inputGwa = inputGwa;
		this.sex = sex;
		this.age = age;
		this.actingDate = actingDate;
		this.comments = comments;
		this.bunho = bunho;
		this.doctor = doctor;
		this.updName = updName;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public Date getReserDate() {
		return reserDate;
	}
	public void setReserDate(Date reserDate) {
		this.reserDate = reserDate;
	}
	public String getReserTime() {
		return reserTime;
	}
	public void setReserTime(String reserTime) {
		this.reserTime = reserTime;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getInputId() {
		return inputId;
	}
	public void setInputId(String inputId) {
		this.inputId = inputId;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getInputGwa() {
		return inputGwa;
	}
	public void setInputGwa(String inputGwa) {
		this.inputGwa = inputGwa;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Integer getAge() {
		return age;
	}
	public void setAge(Integer age) {
		this.age = age;
	}
	public Date getActingDate() {
		return actingDate;
	}
	public void setActingDate(Date actingDate) {
		this.actingDate = actingDate;
	}
	public String getComments() {
		return comments;
	}
	public void setComments(String comments) {
		this.comments = comments;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getUpdName() {
		return updName;
	}
	public void setUpdName(String updName) {
		this.updName = updName;
	}
	
	
	
	
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SchsSCH0201Q04ReserTimeListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
	public class SchsSCH0201Q04ReserTimeListInfo
	{
		private String _hangmogCode;
		private String _hangmogName;
		private String _reserDate;
		private String _reserTime;
		private String _suname;
		private String _inputId;
		private String _inOutGubun;
		private String _inputGwa;
		private String _sex;
		private String _age;
		private String _actingDate;
		private String _comments;
		private String _bunho;
		private String _doctor;
		private String _updName;

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String HangmogName
		{
			get { return this._hangmogName; }
			set { this._hangmogName = value; }
		}

		public String ReserDate
		{
			get { return this._reserDate; }
			set { this._reserDate = value; }
		}

		public String ReserTime
		{
			get { return this._reserTime; }
			set { this._reserTime = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String InputId
		{
			get { return this._inputId; }
			set { this._inputId = value; }
		}

		public String InOutGubun
		{
			get { return this._inOutGubun; }
			set { this._inOutGubun = value; }
		}

		public String InputGwa
		{
			get { return this._inputGwa; }
			set { this._inputGwa = value; }
		}

		public String Sex
		{
			get { return this._sex; }
			set { this._sex = value; }
		}

		public String Age
		{
			get { return this._age; }
			set { this._age = value; }
		}

		public String ActingDate
		{
			get { return this._actingDate; }
			set { this._actingDate = value; }
		}

		public String Comments
		{
			get { return this._comments; }
			set { this._comments = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String UpdName
		{
			get { return this._updName; }
			set { this._updName = value; }
		}

		public SchsSCH0201Q04ReserTimeListInfo() { }

		public SchsSCH0201Q04ReserTimeListInfo(String hangmogCode, String hangmogName, String reserDate, String reserTime, String suname, String inputId, String inOutGubun, String inputGwa, String sex, String age, String actingDate, String comments, String bunho, String doctor, String updName)
		{
			this._hangmogCode = hangmogCode;
			this._hangmogName = hangmogName;
			this._reserDate = reserDate;
			this._reserTime = reserTime;
			this._suname = suname;
			this._inputId = inputId;
			this._inOutGubun = inOutGubun;
			this._inputGwa = inputGwa;
			this._sex = sex;
			this._age = age;
			this._actingDate = actingDate;
			this._comments = comments;
			this._bunho = bunho;
			this._doctor = doctor;
			this._updName = updName;
		}

	}
}